rest/handler.go

```diff
@@ -442,11 +456,10 @@ func (h *handler) invoke(method handlerMethod, accessPermissions []Permission, r
 	// Now set the request's Database (i.e. context + user)
 	if dbContext != nil {
 		h.keyspaceScope, h.keyspaceCollection = *keyspaceScope, *keyspaceCollection
-		h.db, err = db.GetDatabase(dbContext, h.user)
+		h.db, err = db.GetDatabase(h.ctx(), dbContext, h.user)
```

Comment:
db.GetDatabase is going to call dbCtx.AddDatabaseLogContext(ctx), but this has already been done above on line 271.  I think we want to avoid wrapping the context multiple times for the same logging context, do you agree?

Reply:
agreed, let me see if I can remove LogContext from GetDatabase, thanks.